app/feats/direct_entitlement_grant_feat.py

```
@@ -0,0 +1,280 @@
+            correlation_id="",
+            quantity_granted=0,
+            error_code="POLICY_NOT_FOUND",
+            error_message=f"Policy UUID {policy_uuid} not found (may have been deleted)",
+        )
+    except (PolicyParseError, PolicyValidationError) as e:
+        return DirectGrantResult(
+            success=False,
+            correlation_id="",
+            quantity_granted=0,
+            error_code="POLICY_INVALID",
+            error_message=f"Policy validation failed: {str(e)}",
```

Ruff (0.16.0)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)